lib/commands/releases.ts

```diff
@@ -0,0 +1,89 @@
+/**
+ * @license
+ * Copyright 2016-2020 Balena Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { flags } from '@oclif/command';
+import Command from '../command';
+import * as cf from '../utils/common-flags';
+import { getBalenaSdk, getVisuals, stripIndent } from '../utils/lazy';
+import { applicationNameNote } from '../utils/messages';
+import type * as BalenaSdk from 'balena-sdk';
+
+interface FlagsDef {
+	help: void;
+}
+
+interface ArgsDef {
+	fleet?: string;
+}
+
+export default class ReleasesCmd extends Command {
+	public static description = stripIndent`
+		List all releases of a fleet.
+
+		List all releases of the given fleet.
+
+		${applicationNameNote.split('\n').join('\n\t\t')}
```

Comment:
I am contributing a commit that expands some descriptions. Here's how the help output looks like:
```
$ ./bin/balena help release finalize
Finalize a release.

USAGE
  $ balena finalize <commitOrId>

ARGUMENTS
  [commitOrId]  the commit or ID of the release to get information

OPTIONS
  -h, --help  show CLI help

DESCRIPTION
  Finalize a release. Releases can be "draft" or "final", and this command
  changes a draft release into a final release. Draft releases can be created
  with the `--draft` option of the `balena build` or `balena deploy`
  commands.

  Draft releases are not automatically deployed to "unpinned" devices, i.e.
  devices that track the latest available release. For a draft release to
  be deployed to a device, the release should be explicity pinned to that
  device. Conversely, final releases may trigger immediate deployment to
  unpinned devices (subject to a device's  polling period) and, for this
  reason, final releases cannot be changed back to draft status.

EXAMPLES
  $ balena release finalize a777f7345fe3d655c1c981aa642e5555
  $ balena release finalize 1234567
```

Reply:
wow, really good.